### DISCUSSION TITLE
Make video windows work with the Fancyzones utility for Windows

### DISCUSSION THREAD

**Discussion by ashwin911** [f1multiviewer/issue-tracker#152] [Feature requests] [answered]
Fancyzones utility compatibility would make it a lot easier to scale and place the video windows to a predetermined area on the screen, than with the current built-in options.

With Firefox when I pop out the videos on the F1TV website the Fancyzones tool works and detects the shift + drag command when I drag the video window around. But with Multiviewer nothing happens when I shift + drag the video window. Only the main menu window works. Would be really great if it can be fixed.

thanks.

**Comment by katepolak** [comment]
Very much second this. It's an essential feature because of my monitor setup, which renders the built-in layout features completely useless, and the app as a whole completely unusable for me.

**Comment by JJWatMyself** [comment]
Yeah, this would be great.  I think the reason this doesn't work is because there is no title bar, therefore shift drag doesn't activate the feature.  But as a workaround, you can press windows left arrow and FacnyZones will snap the video window to the zone.  Another option is FancyZones itself to add this and can be requested.  It will then work for more software too, not just MV.

Here is the video walk-through I did.

https://discord.com/channels/982375740764983366/1012990702793269328/1034209740579934220
or directly at https://photos.app.goo.gl/sE4xCwvnyKSsP2KZ8

mod edit: add google photos link

**Comment by katepolak** [comment, marked as answer]
I do have to apologise, for some reason even the arrow keys didn't work when I tried it before posting the comment, but they do work now. Which makes it usable enough.
